nautobot/extras/models/models.py

```diff
@@ -112,6 +118,9 @@ class ConfigContext(BaseModel, ChangeLoggedModel, ConfigContextSchemaValidationM
     tenant_groups = models.ManyToManyField(to="tenancy.TenantGroup", related_name="+", blank=True)
     tenants = models.ManyToManyField(to="tenancy.Tenant", related_name="+", blank=True)
     tags = models.ManyToManyField(to="extras.Tag", related_name="+", blank=True)
+    dynamic_groups = models.ManyToManyField(
```

Comment:
Should we have a comment in the code here noting that this field will be empty unless the config feature is enabled?

Reply:
I will add this. Thanks for the reminder.